content/about/index.md

```diff
@@ -47,10 +59,10 @@ PyBaMM is a Sponsored Project of NumFOCUS, a 501(c)(3) nonprofit charity in the
 
 The PyBaMM Steering Council is responsible for the overall direction of the project. The Steering Council is made up of the following members:
 
-- [Ferran Brosa Planella](https://www.brosaplanella.xyz)
-- [Saransh Chopra](https://saransh-cpp.github.io)
-- Scott Marquis
-- [Gregory Offer](https://www.imperial.ac.uk/people/gregory.offer)
-- [Valentin Sulzer](https://sites.google.com/view/valentinsulzer)
+- [Ferran Brosa Planella](https://www.brosaplanella.xyz) - University of Warwick and Ionworks
+- [Saransh Chopra](https://saransh-cpp.github.io) -
```

Comment:
@Saransh-cpp which affiliation(s) do you want here?

Reply:
"Cluster Innovation Centre, University of Delhi" should be good 🙂